TeamSix/pacman.py

```diff
@@ -75,6 +85,52 @@ def handle_key_events(self):
                     self.new_direction = POSITION_RIGHT
 
 
+class Coins:
+    def __init__(self, number_of_coins):
+        self.positions = []
+        self.number_of_coins = number_of_coins
+
+    def generate(self):
+        cnt = 0
+        position_x = 80
+        position_y = 50
+        position_x_tmp = position_x
+        position_y_tmp = position_y
+        self.positions.append((position_x, position_y))
+
+        while len(self.positions) < self.number_of_coins:
+            if cnt % 3 == 0:
+                rnd_num = random.randint(0, 3)
+            cnt += 1
+            if rnd_num == 0:
+                position_x += 30
+            elif rnd_num == 1:
+                position_y += 30
+            elif rnd_num == 2:
+                position_x -= 30
+            elif rnd_num == 3:
+                position_y -= 30
+            if position_y > 900 or position_y < 0 or position_x > 900 or position_x < 0:
```

Comment:
I suppose the 900 represents the screen size here if so, we already have a constant declared for that.